settings.php

```diff
@@ -38,6 +38,11 @@
         get_string('config_delay_duration_desc', 'tool_lifecycle'),
         183 * 24 * 60 * 60)); // Dafault value is 180 days.
 
+    $settings->add(new admin_setting_configtext('tool_lifecycle/backup_path',
```

Comment:
This could be an `admin_setting_configdirectory` instead, which provides some sort of feedback as to whether the directory exists.

Reply:
... in that context, have a look at how it is used in https://github.com/moodle/moodle/blob/8cf4c97217230e74e57e41f24e8afbbc8393eb79/admin/settings/analytics.php#L134-L142, which includes auto-creation of the default path.

Reply:
I am against autocreation. But the type is changed.

Reply:
Also found the old reference and fixed it.